tools/st2_deploy.sh

```diff
@@ -97,6 +99,22 @@ else
     MISTRAL_STABLE_BRANCH="st2-0.5.1"
 fi
 
+update_hosts_file()
+{
+  if [ -z "`cat /etc/hosts | grep -e '127.0.0.1\slocalhost'`" ]; then
```

Comment:
IIUC, this basically sets localhost to 127.0.0.1. Why can't you edit to configuration for the service (postgres in this) on the box and replace localhost with whatever IP we want?